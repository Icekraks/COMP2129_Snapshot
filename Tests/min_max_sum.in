set a 1 2 3
set b 4 5 6
min a
max b
list keys
len a
len b
del a
sum b
bye
