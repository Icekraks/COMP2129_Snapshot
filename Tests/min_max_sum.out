ok
ok
1
6
b
a
3
3
ok
15
bye
